feign-reactor-test/feign-reactor-server-configurations/src/test/java/reactivefeign/wiremock/WireMockServerConfigurations.java

```diff
@@ -25,19 +30,22 @@ protected ServerConnector createHttpConnector(
                                     NetworkTrafficListener listener) {
 
                                 HttpConfiguration httpConfig = createHttpConfig(jettySettings);
+
                                 HTTP2CServerConnectionFactory http2CFactory = new HTTP2CServerConnectionFactory(httpConfig);
-                                http2CFactory.setMaxConcurrentStreams(CALLS_NUMBER);
+                                http2CFactory.setMaxConcurrentStreams(maxConcurrentCalls);
 
                                 return createServerConnector(
                                         bindAddress,
                                         jettySettings,
                                         port,
                                         listener,
-                                        new HttpConnectionFactory(httpConfig),
-                                        http2CFactory
+                                        includeH1
```

Comment:
why not always include http1  and drop boolean input parameter?

Reply:
jetty may work directly with h2c
java11 needs http1 to make upgrade